Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+  os:
+    name: iOS
+    version: "10.3"
+    platform: ""
+  client:
+    type: mobile app
+    name: LBC
+    version: "1544724632"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Player/LG Player 1.0 for Android 9 (stagefright alternative)'
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: mobile app
+    name: LG Player
+    version: "1.0"
+  device:
+    type: smartphone
+    brand: LG
+    model: 'Player'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'LG-H870/V20a Player/LG Player 1.0 for Android 8.0.0 (stagefright alternative)'
+  os:
+    name: Android
+    version: "8.0.0"
+    platform: ""
+  client:
+    type: mobile app
+    name: LG Player
+    version: "1.0"
+  device:
+    type: smartphone
+    brand: LG
+    model: 'G6'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Listen/1 CFNetwork/1107.1 Darwin/19.0.0'
+  os:
+    name: iOS
+    version: "13.0"
+    platform: ""
+  client:
+    type: mobile app
+    name: Listen
+    version: "1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Listen App/84 CFNetwork/1240.0.4 Darwin/20.6.0'
+  os:
+    name: iOS
+    version: "14.7"
+    platform: ""
+  client:
+    type: mobile app
+    name: Listen
+    version: "84"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Listen%20App/136 CFNetwork/1240.0.4 Darwin/20.6.0'
+  os:
+    name: iOS
+    version: "14.7"
+    platform: ""
+  client:
+    type: mobile app
+    name: Listen
+    version: "136"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Liulo/1.1.4 (iPhone; iOS 12.5.6; Scale/2.00)'
+  os:
+    name: iOS
+    version: "12.5.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: Liulo
+    version: "1.1.4"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Liulo/1.1.6 (vn.bachminh.podcastvn; build:6; iOS 17.0.0) Alamofire/5.6.4'
+  os:
+    name: iOS
+    version: "17.0.0"
+    platform: ""
+  client:
+    type: mobile app
+    name: Liulo
+    version: "1.1.6"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Liulo/6 CFNetwork/1331.0.7 Darwin/21.4.0'
+  os:
+    name: iOS
+    version: "15.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: Liulo
+    version: "6"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Listen5 1.2 build 36 (iPhone; iOS 16.5.1; en_US)'
+  os:
+    name: iOS
+    version: "16.5.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Listen5
```

Comment:
Based on my googling, this is https://apptopia.com/ios/app/1444970658/about (bundle has "listen5" in it), so this and other entries probably should be "Just Listen" instead of "Listen 5"